pkgs/development/python-modules/uvloop/default.nix

```diff
@@ -34,8 +34,10 @@ buildPythonPackage rec {
   checkInputs = [
     aiohttp
     pytestCheckHook
-    pyopenssl
     psutil
+  ] ++ lib.optionals (!(stdenv.isDarwin && stdenv.isAarch64)) [
+    # test dependency which is broken on aarch64-darwin
+    pyopenssl
```

Comment:
How much test coverage would be removed if we just disable it on all platforms? I would be fine with that because then we don't have the extra maintenance burden of having platform specific tests.

Reply:
The project is compiled with cython, my attempt to run coverage on .pyx files failed with the error, no time to debug it.

```bash
$ cat .coveragerc
[run]
plugins = Cython.Coverage

$ pip install -e .[dev]
$ pip install coverage
$ coverage --version
Coverage.py, version 6.4.1 with C extension
Full documentation is at https://coverage.readthedocs.io

$ make debug
$ PYTHON="coverage run" make test
$ coverage report

No source for code: '/Projects/uvloop/stringsource'.
```

Reply:
How about we just drop it? I don't want to manage the extra maintenance by having platform specific checks which are not well supported by upstream.

Reply:
done